modules/apps/27-interchain-accounts/controller/keeper/handshake_test.go

```diff
@@ -32,6 +33,13 @@ func (suite *KeeperTestSuite) TestOnChanOpenInit() {
 			func() {},
 			nil,
 		},
+		{
+			"success: empty host connection ID",
+			func() {
+				path.EndpointB.ConnectionID = ""
```

Comment:
What is this test case actually testing?

Reply:
This test is for my old code. I removed it. Thank you sir